raiden/ui/cli.py

```diff
@@ -275,12 +275,12 @@ def run(ctx, **kwargs):
 
         app_ = ctx.invoke(app, **kwargs)
 
-        app_.discovery.register(
+#         spawn address registration to avoid block while waiting for the next block
```

Comment:
if you fix this comment by moving the '#' to the correct position the flake8 check would pass